SETUP.md

```
@@ -34,104 +33,36 @@ There are several ways to run your own instance of CircuitVerse:
 Before making a pull request, it is a good idea to check that all tests are passing locally.
 
-- To run the system tests, run `bundle exec rspec` .
-- To run the simulator tests, run `yarn run test` .
+- To run the system tests, run `bundle exec rspec`.
+- To run the simulator tests, run `yarn run test`.
 
 **Note:** To pass the system tests, you need the [Chrome Browser](https://www.google.com/chrome/) installed. For docker based setup, you can ignore this.
 
-
 ## CircuitVerse API documentation
 CircuitVerse API documentation is available at - https://api.circuitverse.org/
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)